subworkflows/local/germline_variant_calling.nf

```diff
@@ -94,6 +96,17 @@ workflow GERMLINE_VARIANT_CALLING {
         haplotypecaller_vcf = CONCAT_HAPLOTYPECALLER.out.vcf
     }
 
+    if ('deepvariant' in params.tools.toLowerCase()) {
```

Comment:
@maxulysse , please correct me if I'm wrong in assuming that this is how I could integrate the WIP module?

Reply:
That's perfect